src/components/index.ts

```diff
@@ -0,0 +1,4 @@
+// external components
+export { CourtOrderPoa } from '@bcrs-shared-components/court-order-poa'
+export { DatePicker } from '@bcrs-shared-components/date-picker'
+export { EffectiveDateTime } from '@bcrs-shared-components/effective-date-time'
```

Comment:
This is so that components in this app can import shared components the same was as local components. This is particulaly so that EffectiveDateTime can import DatePicker, both in the shared repo (ie, for tests + Storybook) and in this app.